### ISSUE TITLE
Overscroll spring effect at end of pages

### ISSUE THREAD

**Issue by lucidNTR** [brave/browser-laptop#4520]
## Test plan
1. Launch Brave on macOS
2. Load a https://brave.com and scroll down to end of page
3. Keep trying to scroll more (try using mouse and trackpad)
4. Browser should not have a "spring" effect where it moves down and then back up. It should just not scroll

## Original issue details
**Did you search for similar issues before submitting this one?**
yes

**Describe the issue you encountered:**
scrolling to the end of a page feels a bit rough if you are used to spring effects from safari and chrome. we should have overscroll spring effects with the theme color in the background at some some time probably version 2 feature

**Expected behavior:**
overscroll effects.
- Platform (Win7, 8, 10? macOS? Linux distro?):
  all
- Brave Version:
  current master
- Steps to reproduce:
  1. scroll to end of page and scroll further
  2. page just stops moving

**Comment by Liunkae** [comment]
Scroll elasticity would definitely provide a more native feel on macOS (and other platforms too, I guess).

**Comment by bsclifton** [comment]
I believe Electron has a setting for this which gets passed when creating BrowserWindow:
https://github.com/electron/electron/blob/master/docs/api/browser-window.md

scrollBounce

**Comment by Liunkae** [comment]
This is working correctly now. Not sure when it changed. Tested with Brave 0.19.131 on macOS 10.13.

**Comment by bsclifton** [comment]
Closing so this can be tested / confirmed as fixed

**Comment by LaurenWags** [comment]
Reproduced 'spring effect' using trackpad:
![4520](https://user-images.githubusercontent.com/28145373/35070022-0055a3fe-fbaa-11e7-8682-907fc798cb3a.gif)

**Comment by petemill** [comment]
This issue description seems to say _both_ that there should be a spring effect, and there should not be a sprint effect. My interpretation is that the issue is saying there should be the effect, and my intuition is that there should be the effect just like other browsers and apps on macOS. And since the effect is there, I would say this issue is ok and can be closed.

**Comment by LaurenWags** [comment]
@petemill - should I be seeing a difference with trackpad vs mouse? because I only see the spring effect with trackpad, not with mouse.

**Comment by petemill** [comment]
yes @LaurenWags - it is expected across the OS that the overscroll spring is only experienced with touch input on trackpad

**Comment by bsclifton** [comment]
@petemill ah- good to know 😄  I would be 👍  on closing issue as won't fix and removing milestone

**Comment by Liunkae** [comment]
Just for the record, Brave used to not have the spring effect. Now it does, which is the expected behavior for macOS apps and the trackpad.
